### ISSUE TITLE
Running elm-test after elm-verify-examples fails

### ISSUE THREAD

**Issue by zwilias** [stoeffel/elm-verify-examples#74]
- elm-test@0.19.0-beta4
- elm-verify-examples@3.0.0

Running `elm-test` after running `elm-verify-examples` fails with an error:

```
Received invalid JSON from test interface search: SyntaxError: Unexpected token n JSON at position 0
```

The data it's trying to parse as JSON is something like this:
```
elmi-to-json failed for:
all

I didn't find a module for the interface /Users/ilias/Src/elm/elm-rosetree/elm-stuff/generated-code/elm-explorations/test/elm-stuff/0.19.0/VerifyExamples-Tree-ReplaceLabel0.elmi.
```

It appears that elm-verify-examples removes the generated code but not the interface files. `elm-test` then runs `elmi-to-json`, which tries to lookup the matching modules.

**Comment by stoeffel** [comment]
cc @rtfeldman. The fix should be in elmi-to-json. It shouldn't fail on elmis if the corresponding source got removed.

**Comment by stoeffel** [comment]
cc @rtfeldman. The fix should be in elmi-to-json. It shouldn't fail on elmis if the corresponding source got removed.

**Comment by stoeffel** [comment]
Fix in elmi-to-json got published elmi-to-json@0.19.0. thanks to @zwilias ❤️

**Comment by stoeffel** [comment]
this should be resolved once elm-test uses the latest version of elmi-to-json
